doc/releases/nautilus.rst

```diff
@@ -5,6 +5,144 @@ Nautilus
 Nautilus is the 14th stable release of Ceph.  It is named after the
 nautilus, a family of cephalopods characterized by a whorled shell.
 
+v14.2.22 Nautilus
+=================
+
+
```

Comment:
Should mention that this is likely the last Nautilus release.

Reply:
@idryomov what would I say for `released` date in `doc/releases/releases.yml` ?

Reply:
`Should mention that this is likely the last Nautilus release` done